src/core/token/TokenSpawnTracker.ttslua

```diff
@@ -39,31 +28,20 @@ function resetTokensSpawned(cardGuid)
   spawnedCardGuids[cardGuid] = nil
 end
 
-function resetAllAssetAndEvents()
```

Comment:
The changes here also make it so putting a card into a hand zone no longer resets its ability to spawn tokens